package.json

```diff
@@ -79,7 +79,7 @@
     "@leafygreen-ui/menu": "18.0.0",
     "@leafygreen-ui/modal": "6.1.2",
     "@leafygreen-ui/palette": "3.4.4",
-    "@leafygreen-ui/popover": "7.2.2",
+    "@leafygreen-ui/popover": "11.0.0",
     "@leafygreen-ui/radio-box-group": "12.0.0",
     "@leafygreen-ui/radio-group": "10.0.0",
```

Comment:
Dark purple doesn't look too good inside the tooltip, sadly... we can revisit it later 😔
<img width="184" alt="Screen Shot 2022-11-18 at 4 57 16 PM" src="https://user-images.githubusercontent.com/47064971/202810574-e610b3dd-aa2b-4f59-aea6-8bf260058515.png">

Reply:
Good catch—a lot of these icons actually get pretty messed up with the dark background:
<img width="182" alt="image" src="https://user-images.githubusercontent.com/9298431/204308642-4a9ddc24-f8f5-4e37-ba32-36e698705f7e.png">
I'm going to apply dark mode here as a workaround
<img width="187" alt="image" src="https://user-images.githubusercontent.com/9298431/204308892-1d6ecfb8-dc01-4aaa-9f1b-e11a86a17964.png">